release notes/v0.27.0.md

```diff
@@ -0,0 +1,137 @@
+
+Additionally, besides the 4 "old" executor types, there are 3 new executors, added to support some of the most frequently requested load testing scenarios that were previously difficult or impossible to model in k6:
+
+- `per-vu-iterations`: each VU executes a fixed number of iterations ([#381](https://github.com/loadimpact/k6/issues/381)).
+- `constant-arrival-rate`: iterations are started at a specified fixed rate, for a specified duration. This allows k6 to dynamically change the amount of VUs during a test run to achieve the specified amount of iterations per period, which is useful for a more accurate representation of RPS (requests per second), for example. See [#550](https://github.com/loadimpact/k6/issues/550) for details.
+- `ramping-arrival-rate`: a variable number of iterations are executed in a specified period of time. This is similar to the ramping VUs executor, but instead of specifying how many VUs should loop through the script at any given point in time, the iterations per second k6 should execute at that point in time can be specified.
+
+It's important to also note that all of these executors, except the `externally-controlled` one, can be used both in local k6 execution with `k6 run`, and in the distributed cloud execution with `k6 cloud`. This even includes "old" executors that were previously unavailable in the cloud, like the `shared-iterations` one. Now, you can execute something like `k6 cloud --iterations 10000 --vus 100 script.js` without any issues.
+
+#### Execution improvements
+
+Multiple execution scenarios can now be configured in a single test run via the new `scenarios` option. These scenarios can run both sequentially and in parallel, and can independently execute different script functions, have different executor types and execution options, and have custom environment variables and metrics tags.
+
+An example using 2 scenarios:
+
+```js
+import http from 'k6/http';
+import { fail } from 'k6';
+
+export let options = {
+  discardResponseBodies: true,
+  scenarios: {
+    my_web_test: {
+      executor: 'constant-vus',
+      exec: 'webtest',
+      vus: 50,
+      duration: '30s',
+      tags: { my_test_version: 'v2' },
+      env: { TESTVERSION: 'v2' },
+    },
+    my_api_test: {
+      executor: 'ramping-arrival-rate',
+      exec: 'apitest',
+      startTime: '30s',
+      startRate: 50,
+      timeUnit: '1s',
+      preAllocatedVUs: 50,
+      maxVUs: 100,
+      stages: [
+        { target: 200, duration: '30s' },
+        { target: 0, duration: '30s' },
+      ],
+      tags: { my_test_version: 'v1' },
+      env: { TESTVERSION: 'v1' },
+    },
+  }
+};
+
+export function webtest() {
+  if (__ENV.TESTVERSION != 'v2') fail();
+  http.get('https://test.k6.io/contacts.php');
+}
+
+export function apitest() {
+  if (__ENV.TESTVERSION != 'v1') fail();
+  http.get('https://test-api.k6.io/');
+}
```

Comment:
I think `fail()` will only confuse people here and is superfluous 

```suggestion
import http from 'k6/http';
import { sleep } from 'k6';

export let options = {
    scenarios: {
        my_web_test: { // some arbitrary scenario name
            executor: 'constant-vus',
            vus: 50,
            duration: '5m',
            gracefulStop: "0s", // do not wait for iterations to finish in the end
            tags: { test_type: 'website' }, // extra tags for the metrics generated by this scenario
            exec: 'webtest', // the function this scenario will execute
        },
        my_api_test_1: {
            executor: 'constant-arrival-rate',
            rate: 90, timeUnit: '1m', // 90 iterations per minute, i.e. 1.5 RPS
            duration: '5m',
            preAllocatedVUs: 10, // the size of the VU (i.e. worker) pool for this scenario

            tags: { test_type: 'api' }, // different extra metric tags for this scenario
            env: { MY_CROC_ID: '1' }, // and we can specify extra environment variables as well!
            exec: 'apitest', // this scenario is executing different code than the one above!
        },
        my_api_test_2: {
            executor: 'ramping-arrival-rate',
            startTime: '30s', // the ramping API test starts a little later
            startRate: 50, timeUnit: '1s', // we start at 50 iterations per second
            stages: [
                { target: 200, duration: '30s' }, // go from 50 to 200 iters/s in the first 30 seconds
                { target: 200, duration: '3m30s' }, // hold at 200 iters/s for 3.5 minutes
                { target: 0, duration: '30s' }, // ramp down back to 0 iters/s over the last 30 second
            ],
            preAllocatedVUs: 50, // how large the initial pool of VUs would be
            maxVUs: 100, // if the preAllocatedVUs are not enough, we can initialize more

            tags: { test_type: 'api' }, // different extra metric tags for this scenario
            env: { MY_CROC_ID: '2' }, // same function, different environment variables
            exec: 'apitest', // this scenario is executing different code than the one above!
        },
    },
    discardResponseBodies: true,
    thresholds: {
        // we can set different thresholds for the different scenarios because
        // of the extra metric tags we set!
        "http_req_duration{test_type:api}": ["p(95)<250", "p(99)<350"],
        "http_req_duration{test_type:website}": ["p(99)<500"],
        // we can reference the scenario names as well
        "http_req_duration{scenario:my_api_test_2}": ["p(99)<300"],
    }
};

export function webtest() {
    http.get('https://test.k6.io/contacts.php');
    sleep(Math.random() * 2);
}

export function apitest() {
    http.get(`https://test-api.k6.io/public/crocodiles/${__ENV.MY_CROC_ID}/`);
    // no need for sleep() here, the iteration pacing will be controlled by the
    // arrival-rate executors above!
}
```